.github/workflows/tenzir.yaml

```
@@ -348,9 +348,14 @@ jobs:
         run: |
           git submodule update --init contrib/tenzir-plugins
           git -C contrib/tenzir-plugins fetch origin main
+          # Check that the plugins submodule commit is either already merged
+          # into `main` or branches off directly from current `main`.
```

The wording "branches off directly from current main" suggests that the commit that the submodule is pointing to does not include any fork points with other HEADs after the merge-base with main.
I suggest to change the phrase:
```suggestion
          # Check that the plugins submodule commit is either already merged
          # into `main` or descends from current `main`.
```